src/main/java/hu/icellmobilsoft/atr/sample/rest/parseJson.java

```diff
@@ -0,0 +1,195 @@
+package hu.icellmobilsoft.atr.sample.rest;

```

Comment:
Ez helper/sehítő osztály logika, mozgassuk át a util package-be kerülön egy ParseHelper osztályba. Classok egyéblént kezdődjenek mindig nagy betűvel. Rest packageben csak interface-ek és azok implementálása (Impl végű classok) szerepeljenek.